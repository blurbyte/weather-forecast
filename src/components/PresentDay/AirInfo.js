import React from 'react';
import PropTypes from 'prop-types';

import { removeDecimals } from '../../utils';
import Row from '../Row';
import InfoText from './InfoText';
import Label from './Label';

const AirInfo = ({ wind, humidity, pressure }) => (
  <Row>
    <InfoText>
      <Label>Wind</Label> {removeDecimals(wind)} m/s
    </InfoText>
    <InfoText>
      <Label>Humidity</Label> {humidity} %
    </InfoText>
    <InfoText>
      <Label>Pressure</Label> {removeDecimals(pressure)} hpa
    </InfoText>
  </Row>
);

AirInfo.propTypes = {
  wind: PropTypes.number.isRequired,
  humidity: PropTypes.number.isRequired,
  pressure: PropTypes.number.isRequired
};

export default AirInfo;
