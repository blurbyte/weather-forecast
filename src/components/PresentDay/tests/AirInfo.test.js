import React from 'react';
import renderer from 'react-test-renderer';

import AirInfo from '../AirInfo';

test('renders correctly', () => {
  const component = renderer.create(<AirInfo />);
  expect(component.toJSON()).toMatchSnapshot();
});
