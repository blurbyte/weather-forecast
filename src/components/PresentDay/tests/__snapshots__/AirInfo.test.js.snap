// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa iInree"
>
  <p
    className="sc-htpNat jsqgjC sc-bwzfXH dcdldk"
  >
    <span
      className="sc-bxivhb iBbKEM"
    >
      Wind
    </span>
     5.1 m/s
  </p>
  <p
    className="sc-htpNat jsqgjC sc-bwzfXH dcdldk"
  >
    <span
      className="sc-bxivhb iBbKEM"
    >
      Humidity
    </span>
     42 %
  </p>
  <p
    className="sc-htpNat jsqgjC sc-bwzfXH dcdldk"
  >
    <span
      className="sc-bxivhb iBbKEM"
    >
      Pressure
    </span>
     1012 hpa
  </p>
</div>
`;
